layouts/index.html

```diff
@@ -0,0 +1,128 @@
+{{ define "main" }}
+  <div class="container p-3 py-md-5">
+    <h1>SRCF Documentation</h1>
+    <p class="fs-5 col-md-8">
+      Hello there! This is the Student-Run Computing Facility's documentation.
+      Our revamped docs contain all manner of useful pages, such as getting
+      started guides, tutorials and a technical reference. In this revision
+      we've placed an emphasis on learning, tutorials and making information
+      more technically accessible so that you can fully leverage our services.
+    </p>
+
+    <p class="fs-5 col-md-8">
+      You can navigate the docs by exploring our links below, using our search,
+      or <a href="docs.pdf">downloading a single PDF</a> at your leisure.
+    </p>
+
+    <hr class="col-3 col-md-2 mb-5 mt-3 mt-md-5" />
+
+    <div class="row g-5 row-cols-md-2">
+      <div class="col">
+        {{ $heading := "Reference" }}
+        <h2>{{ $heading }}</h2>
+        <p>
+          A detailed technical breakdown of our services, servers and overall
+          set-up. If you want to know more about a particular aspect of the
+          SRCF, this is the place.
+        </p>
+        <ul class="fa-ul">
+          {{ $highlights := where .Site.Pages "Section" ($heading | urlize) }}
+
+          {{ range $highlights }}
+            {{ if (eq .Params.highlight true) }}
+              {{ partial "lists/fa-link" (dict "ctx" . "fa" "fas fa-arrow-circle-right") }}
+            {{ end }}
+          {{ end }}
+
+          {{ partial "lists/fa-text" (dict "fa" "fas fa-arrow-circle-right" "text" "...and more!") }}
+        </ul>
+        <a class="btn btn-secondary" role="button" href="/reference"
```

Comment:
The links on this page assume a site root for the base URL -- as this is the index page, we can just do away with the leading slashes and link to `reference/`, `tutorials/` etc.

Reply:
Done!